data/seeds/01-projects.js

```diff
@@ -1 +1,14 @@
 
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
```

Comment:
Great job using `// ...` to explain what's happening in the code section. Keep that up!